src/Sentry.Maui/BindableSentryMauiOptions.cs

```diff
@@ -9,6 +9,7 @@ internal class BindableSentryMauiOptions : BindableSentryLoggingOptions
     public bool? IncludeTitleInBreadcrumbs { get; set; }
     public bool? IncludeBackgroundingStateInBreadcrumbs { get; set; }
     public bool? CreateElementEventsBreadcrumbs { get; set; } = false;
+    public bool? AttachScreenshots { get; set; } = false;
```

Comment:
If we default to `false`, why is it nullable? @jamescrosswell might know since he did this bindable bits?

Reply:
Removed the default value.

Reply:
The value above has a default, should this have a default? Should we remove it everywhere (where is default source of truth)?

Still hoping @jamescrosswell can chime in he might know these answers